cli/dcoscli/tables.py

```diff
@@ -441,20 +462,328 @@ def id_and_containers(pod):
         container_lines = ('\n |-{}'.format(name) for name in container_names)
         return pod_id + ''.join(container_lines)
 
+    def is_overdue(row):
```

Comment:
Refactored this to enhance the input table data. See main.py#_enhance_row_with_overdue_information